### ISSUE TITLE
non-CX compatibility

### ISSUE THREAD

**Issue by critor** [zephray/cpuid-nspire#1]
Hi.

Your 1.0 release doesn't seem to work on (all?) non-color TI-Nspire.
Just tested on 2 different TI-Nspire ClickPad (datestamp AS-0507 and DVT1.2), cpuid just shows a blank screen.

I currently don't have access to more recent TI-Nspire ClickPad or TouchPad.

**Comment by zephray** [comment]
Yes just tested on my side... the version I released on GitHub doesn't seem to work. No idea what happened there, but I have a local copy that is known to work with my classic nspires (both CP and TP), can you try this？

[cpuid-nspire.zip](https://github.com/zephray/cpuid-nspire/files/5538353/cpuid-nspire.zip)

**Comment by critor** [comment]
Yes, it works perfectly.
Thank you very much for being so prompt.
